cmd/hubd/main.go

```diff
@@ -116,6 +116,10 @@ var (
 				Key:      "threads.max_number_per_owner",
 				DefValue: 100,
 			},
+			"tokenPowergateAdmin": {
+				Key:      "token.powergate_admin",
```

Comment:
How about "powergate.admin_token"? That way the config file will have a top-level `powergate` section. Then the flag becomes `powergateAdminToken`.